docs/modules/user-guide/partials/proc_using-the-devfile-library.adoc

```diff
@@ -0,0 +1,156 @@
+[id="proc_using-the-devfile-library_{context}"]
+= Using the devfile library
+
+With the devfile library, you can parse any devfile, helping you develop your containerized applications.
```

Comment:
any thoughts on how we can expand this intro statement so users have more of an idea about what they can use the devfile library for? and how the devfile library helps them?

Reply:
Maybe something along the lines of `By using devfiles, developers and development teams can quickly configure, deploy, and maintain consistent development environments, thus making it easy for teams to collaborate across shared projects.`

Reply:
Love it, thanks, Brian! @yangcao77 any thoughts from the developer's point of view?

Reply:
I'm thinking if the word `parse any devfile` is enough.  the following examples include generate/validate which can probably be a side function of "parser", but what about examples on writer.  so probably `parse and write any devfiles` instead?

Reply:
What kind of animal is the devfile library? You can ride almost any four-legged animal, but are we talking about a camel, a donkey, or an alligator? :-) In other words, is this library an SDK, or something else our developer users might be familiar with?